chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -355,6 +355,14 @@ pendingSurveillance.certifiedProductIdNotFound=Unexpected error looking up certi
 
 #user errors
 user.notFound=User not found.
+user.subjectName.maxlength=You have exceeded the max length for the User Name (%s characters). 
+user.fullName.maxlength=You have exceeded the max length for the Full Name (%s characters).
+user.friendlyName.maxlength=You have exceeded the max length for the Friendly Name (%s characters).
+user.title.maxlength=You have exceeded the max length for the Title (%s characters).
+user.email.maxlength=You have exceeded the max length for the Email (%s characters).
+user.phoneNumber.maxlength=You have exceeded the max length for the Phone Number (%s characters).
+user.subjectName.required=Username ('subject name') is required.
```

Comment:
This looks like weird phrasing. The "maxlength" error refers to the "User Name", and I don't think an end use would have any idea what a 'subject name' is

Reply:
I agree, but that was an existing message.

Reply:
The fact that it's an existing message doesn't mean it's right.